common/include/opae/cxx/token.h

```diff
@@ -37,16 +37,30 @@ namespace opae {
 namespace fpga {
 namespace types {
 
+/** Wraps the OPAE fpga_token primitive.
+ * token's are enumerated based on a set of
```

Comment:
There's an apostrophe in tokens. What about changing this to read something like:
`tokens are created from an enumeration operation that uses properties describing an fpga resource`